### DISCUSSION TITLE
File/Folder name conflict

### DISCUSSION THREAD

**Discussion by KyuubiYoru** [hlky/stable-diffusion-webui#599] [General]
I'm running Stable Diffusion on a Linux machine and tried to sync the output folder to my Windows machine using Nextcloud.
The problem is, Linux is case sensitive for Folder names, but windows isn't. So I get sync conflicts on windows if I have a folder for the prompt "Corgi" and "corgi".
Not sure what would be the best way to fix this, maybe make all folders lower case?

**Comment by andzejsp** [comment]
I would add to this also could there be an option to not put the entire prompt into the folder name? Maybe generate a timestamped foldername instead? like to the milisecond so that there is no conflict. After few prompts and moving folders around filepath exceeds the maximum char number and it gets difficult to like drag n drop into different applications to share a pic or upload it..
